packages/FormElement/stories/examples/ListBox.js

```diff
@@ -1,17 +1,28 @@
 import React from "react";
+import L10n from "@paprika/l10n";
 import ListBox from "@paprika/list-box";
 import FormElement from "../../src";
 
-const { Label, Content } = FormElement;
+const { Label, Content, Description } = FormElement;
 
 export default function ListBoxExample() {
-  const optionsArray = ["Black Panther", "Wonder Woman", "Spiderman", "The Incredibles", "Thor", <span>test</span>];
-  const listboxOptions = optionsArray.map(hero => <ListBox.Option key={hero}>{hero}</ListBox.Option>);
+  const optionsArray = ["Catwoman", "Doctor Octopus", "Thanos"];
```

Comment:
Seriously. Catwoman.

Reply:
Batman has all the best villians!

![](https://media.giphy.com/media/g2rnmfyJwKwU0/giphy.gif)